src/components/Footer.tsx

```diff
@@ -0,0 +1,114 @@
+import Link from 'next/link';
+import { Navigation } from '@/enums/navigation.enum';
+import { SOCIAL_PROVIDER_REACT_ICONS } from '@/constants/social-provider-react-icons';
+
+type FooterSectionType = {
```

Comment:
in the project we usually used interface, I think there is type in several places but is there any special purpose with type?

Reply:
Not, really